python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/router_pim_sparse_mode.schema.yml

```diff
@@ -20,16 +20,12 @@ keys:
             description: IPv4 Prefix associated with SSM.
           rp_addresses:
             type: list
-            # This primary_key is inserted to support conversion from wildcard dicts,
-            # but it also stops the user from defining the same RP address multiple times,
-            # which should be allowed. When wildcard dict support is finally removed,
-            # this primary_key should also be removed. TODO: AVD5.0
-            primary_key: address
```

Comment:
I have been thinking a bit on this, and instead of removing the primary_key and setting required: true, let's use the special `allow_duplicate_primary_key`, since it still allows for merging of custom structured config on schema primary key.

Reply:
Updated.